app/src/main/java/org/oppia/android/app/options/LanguageSelectionViewModel.kt

```diff
@@ -12,23 +13,25 @@ import javax.inject.Inject
  */
 @FragmentScope
 class LanguageSelectionViewModel @Inject constructor(
```

Comment:
Can you confirm that there is existing test coverage verifying the whole language selection flow (just to ensure that the latest changes are "correct" per passing tests)?

Reply:
No, the radio button is not functional yet

Reply:
I guess we need to test if the Language string is showing or not 
This is because in https://github.com/oppia/oppia-android/pull/2392/commits/173255a6723fbbbac06a25b800bc4b991280e4e0 this commit (before the recent implementation i.e, using listener ) all the tests were passing but when i tested the app the language were not visible and everything works fine